static/index.js

```diff
@@ -1,13 +1,36 @@
 window.onload = () => {
-  const item1 = document.createElement("li");
-  const item2 = document.createElement("li");
-  const item3 = document.createElement("li");
+  // const item1 = document.createElement("li");
+  // const item2 = document.createElement("li");
+  // const item3 = document.createElement("li");
 
-  item1.innerText = "take garbage out";
-  item2.innerText = "Play video game";
-  item3.innerText = "Take over the world";
+  // item1.innerText = "take garbage out";
+  // item2.innerText = "Play video game";
+  // item3.innerText = "Take over the world";
 
-  const listItems = document.createElement("ul");
-  listItems.append(item1, item2, item3);
-  const vanillaList = document.getElementById("list").appendChild(listItems);
+  // listItems.append(item1, item2, item3);
+  // const vanillaList = document.getElementById("list").appendChild(listItems);
+
+  const items = [
+    { title: "take garbage out", checked: true },
+    { title: "Play video games", checked: false },
+    { title: "Take over the world", checked: true }
+  ];
+  const todoItem = items => {
+    items.checked = document.createElement("input");
```

Comment:
we don't want to modify the data on the item passed in, we want to use it to create HTML elements to represent that data.
create a list item element
create an input of type checkbox and set it's checked property to the value of item.checked
append the checkbox to the list item
create a span and set it's innerText to the value of item.title
append the span to the list item
return the list item element